app/views/assessor_interface/check_personal_informations/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<% content_for :page_title, "Check personal information" %>
+
+<% content_for :back_link_url, assessor_interface_application_form_path(@application_form) %>
+
+<h1 class="govuk-heading-xl">Check personal information</h1>
+
+<%= render "shared/application_form/personal_information_summary" , application_form: @application_form,
+   personal_information: @personal_information, changeable: false %>
```

Comment:
```suggestion
<%= render "shared/application_form/personal_information_summary",
           application_form: @application_form,
           changeable: false %>
```

I'd suggest having the indentation like this (we also don't need to pass a `personal_information` variable).